adding annotation information to set sidecar container's seccompprofiletype (#3220)
dapr/docs: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

Refer to the documentation for the [Dapr operator]({{< ref operator >}}) service for more details on the injector watchdog and how to enable it.

## Configuring seccompProfile for sidecar containers

> By default, Dapr sidecar injector injects a sidecar without any `seccompProfile`. However, to have Dapr sidecar contianer run successfully in a namespace with [Restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) profile, sidecar container needs to have `securityContext.seccompProfile.Type` to not be `nil`. 

Reviewer: ```suggestion
By default, the Dapr sidecar Injector injects a sidecar without any `seccompProfile`. However, to have Dapr sidecar container run successfully in a namespace with [Restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) profile, the sidecar container needs to have `securityContext.seccompProfile.Type` to not be `nil`. 
```

> Refer to [this]({{< ref "arguments-annotations-overview.md" >}}) documentation to set appropriate `seccompProfile` on sidecar container accordingly to which profile it would be running with.

Reviewer: ```suggestion
Refer to [this]({{< ref "arguments-annotations-overview.md" >}}) documentation to set appropriate `seccompProfile` on sidecar container according to which profile it is running with.
```
